tests/test_db_store.py

```diff
@@ -3,6 +3,7 @@
 
 import mock
 import datetime
+from dateutil.tz import tzoffset
```

Comment:
Style guide forbids importing functions directly, so we should either do `from dateutil import tz` or `import dateutil.tz`
https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports